Small bug introduced in 2.2

http://crashes.to/s/084e28f934d

For those without Crashlytics access:

    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.gdg.frisbee.android/org.gdg.frisbee.android.eventseries.TaggedEventSeriesActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setChecked(boolean)' on a null object reference
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2417)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2477)
       at android.app.ActivityThread.-wrap11(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1345)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5415)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:725)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:615)
    Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setChecked(boolean)' on a null object reference
       at org.gdg.frisbee.android.common.GdgNavDrawerActivity.setupDrawerContent(GdgNavDrawerActivity.java:153)
       at org.gdg.frisbee.android.common.GdgNavDrawerActivity.initNavigationDrawer(GdgNavDrawerActivity.java:119)
       at org.gdg.frisbee.android.common.GdgNavDrawerActivity.setContentView(GdgNavDrawerActivity.java:88)
       at org.gdg.frisbee.android.eventseries.TaggedEventSeriesActivity.onCreate(TaggedEventSeriesActivity.java:58)
       at android.app.Activity.performCreate(Activity.java:6162)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2370)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2477)
       at android.app.ActivityThread.-wrap11(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1345)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5415)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:725)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:615)

--- a/app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java
+++ b/app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java
@@ -150,7 +150,8 @@ public abstract class GdgNavDrawerActivity extends GdgActivity {
 
         menu.setGroupCheckable(GROUP_ID, true, true);
 
-        MenuItem selectedItem = menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME));
+        final int selectedDrawerItemId = getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME);
+        final MenuItem selectedItem = menu.findItem(selectedDrawerItemId);
         if (selectedItem != null) {
             selectedItem.setChecked(true);
         }

--- a/app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java
+++ b/app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java
@@ -150,7 +150,10 @@ public abstract class GdgNavDrawerActivity extends GdgActivity {
 
         menu.setGroupCheckable(GROUP_ID, true, true);
 
-        menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME)).setChecked(true);
+        MenuItem selectedItem = menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME));
+        if (selectedItem != null) {
+            selectedItem.setChecked(true);
+        }
 
         navigationView.setNavigationItemSelectedListener(
 

